pages/inspect.tsx

```diff
@@ -124,57 +124,144 @@ const Home: NextPage = () => {
     if (
       !isErc725X &&
       !isErc725YContract &&
-      !isERC1271 &&
-      !isLSP0ERC725Account &&
       !isLSP1UniversalReceiver &&
       !isLSP1UniversalReceiverDelegate &&
-      !isLSP6KeyManager &&
-      !isLSP7DigitalAsset &&
-      !isLSP8IdentifiableDigitalAsset &&
-      !isLSP9Vault
+      !isLSP6KeyManager
     ) {
       return (
         <div className="help is-danger inspect-result">
-          <p>ERC725X: ❌</p>
-          <p>ERC725Y: ❌</p>
-          <p>ERC1271: ❌</p>
-          <p>LSP0ERC725Account: ❌</p>
-          <p>LSP1UniversalReceiver: ❌</p>
-          <p>LSP1UniversalReceiverDelegate: ❌</p>
-          <p>LSP6KeyManager: ❌</p>
-          <p>LSP7DigitalAsset: ❌</p>
-          <p>LSP8IdentifiableDigitalAsset: ❌</p>
-          <p>LSP9Vault: ❌</p>
           <p>
             This address is not a valid ERC725 Profile, nor it is a valid LSP
             contract.
           </p>
-          <p>Please check if the addressis correct.</p>
+          <p>Please check if the address is correct.</p>
         </div>
       );
     }
 
     return (
       <div className="help is-success inspect-result">
-        <p>ERC725X: {isErc725X ? '✅' : '❌'}</p>
-        <p>
-          ERC725Y: {isErc725YContract ? '✅' : '❌'} {isErc725Y_v2 && '(v2.0)'}
+        <a
+          className={
+            isErc725X
+              ? 'button is-link mr-2 mt-2'
+              : 'button is-link is-outlined mr-2 mt-2'
+          }
```

Comment:
```suggestion
          className={`button is-link mr-2 mt-2
            ${!isErc725X && 'is-outlined'}`}
```

this might be easier to read - as you activate only the class that makes sense - the mt and button are common for both cases. so no need to duplicate them in the 2 cases of the if.